scripts/zones/Norg/IDs.lua

```diff
@@ -15,6 +15,7 @@ zones[xi.zone.NORG] =
         ITEM_OBTAINED                = 6412,  -- Obtained: <item>.
         GIL_OBTAINED                 = 6413,  -- Obtained <number> gil.
         KEYITEM_OBTAINED             = 6415,  -- Obtained key item: <keyitem>.
+        YOU_OBTAIN                   = 6418,  -- You obtain <number> <item>!<7F31>
```

Comment:
Remove `<7F31>`  Also, need to verify this is the correct ID

Reply:
The ID is correct for current retail.